README.md

```diff
@@ -17,6 +17,8 @@ npm install --save-dev elm-watch
 
 > ℹ️ You need to install Elm itself separately, in whatever way you prefer ([installer][elm-guide-install], [npm][elm-npm-package], [elm-tooling]).
 
+> ℹ️ Node.js 14 or newer is required as elm-watch makes use of ES2020 language features.
```

Comment:
```suggestion
> ℹ️ Node.js 14 or newer is required.
```

It’s not only due to ES2020 language features. I think this is enough?

Reply:
Of course, you know best!